app/src/main/java/com/io/moviO/network/MoviesAPI.kt

```diff
@@ -0,0 +1,20 @@
+package com.io.moviO.network
+
+import com.io.moviO.common.Constants
+import com.io.moviO.data.MoviesListResources
+import com.io.moviO.data.modelMovie.MovieByIdResources
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface MoviesAPI {
+
+    @GET("/3/movie/top_rated?api_key=${Constants.API_KEY}")
+    suspend fun getMovies(): MoviesListResources
+
+    // @POST()
+    // fun uploadMovie(): Unit
+
+    @GET("/3/movie/{id}?api_key=${Constants.API_KEY}&language=en-US")
```

Comment:
Maybe /3/movie/ can be part of the base url, if it is gong to be used for all the requests

Reply:
I disagree, /3 - sure but /movie/{id} is more readable than just {id} and not knowing where this comes from